Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -2951,18 +2952,17 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
 
     m_declarativeNetRequestDynamicRulesStore = nullptr;
     m_declarativeNetRequestSessionRulesStore = nullptr;
-    m_declarativeNetRequestRuleStore = nullptr;
 }
 
-WKContentRuleListStore *WebExtensionContext::declarativeNetRequestRuleStore()
+String WebExtensionContext::declarativeNetRequestContentRuleListFilePath()
```

Comment:
It would probably be to our advantage to separate the extension parts of this from the non-extension parts of this into two PRs.